web/user.go

```diff
@@ -56,6 +56,21 @@ func (u *UserInfo) GetBool(name string) bool {
 	return b
 }
 
+// GetLastLogin gets the user last login time.
+// Updated_at is a timestamp indicating when the user's profile was last updated/modified.
+// Changes to last_login are considered updates, so most of the time, updated_at will match last_login.
+func (u *UserInfo) GetLastLogin() string {
```

Comment:
I think on the goutils side, I would recommend keeping the names equal. Returning `updated_at` should be called GetUpdatedAt. On the app side, we can name it differently. 

Just following what Eric has told me: Keep goutils as generic as possible and don't put App specific logic in there (such as naming, mappings in this case).

Reply:
Thanks, makes sense!  Updated name to GetUpdatedAt.